feg/protos/BUILD.bazel

```diff
@@ -25,3 +26,61 @@ python_proto_library(
     protos = [":mconfigs_proto"],
     deps = ["//orc8r/protos:common_python_proto"],
 )
+
+proto_library(
+    name = "s6a_proxy_proto",
+    srcs = ["s6a_proxy.proto"],
+)
+
+cpp_proto_library(
+    name = "s6a_proxy_cpp_proto",
+    protos = [":s6a_proxy_proto"],
+)
+
+cpp_grpc_library(
+    name = "s6a_proxy_cpp_grpc",
+    protos = [":s6a_proxy_proto"],
+    deps = [":s6a_proxy_cpp_proto"],
+)
+
+proto_library(
+    name = "csfb_proto",
+    srcs = ["csfb.proto"],
+    deps = ["//orc8r/protos:common_proto"],
+)
+
+cpp_proto_library(
+    name = "csfb_cpp_proto",
+    protos = [":csfb_proto"],
+    deps = ["//orc8r/protos:common_cpp_proto"],
```

Comment:
Similar to question above: for Python this dependency is not needed (but, yes, we have some targets where this pattern is currently used). My understanding: the dependency in csfb_proto suffices, .i.e., in this case protoc will compile also common.proto.